### ISSUE TITLE
failed AVER with SBCL 2.1.5

### ISSUE THREAD

**Issue by nsrahmad** [3b/3bz#12]
Trying to Load McCLIM, 3bz fails with:
```lisp
failed AVER:
    (NOT (EQUAL SB-X86-64-ASM::SIZES '(:DWORD :QWORD)))
This is probably a bug in SBCL itself. (Alternatively, SBCL
might have been corrupted by bad user code, e.g. by an undefined
Lisp operation like (FMAKUNBOUND 'COMPILE), or by stray pointers
from alien code or from unsafe Lisp code; or there might be a
bug in the OS or hardware that SBCL is running on.) If it seems
to be a bug in SBCL itself, the maintainers would like to know
about it. Bug reports are welcome on the SBCL mailing lists,
which you can find at <http://sbcl.sourceforge.net/>.
   [Condition of type SB-INT:BUG]

Restarts:
 0: [RETRY] Retry compiling #<CL-SOURCE-FILE "3bz" "deflate">.
 1: [ACCEPT] Continue, treating compiling #<CL-SOURCE-FILE "3bz" "deflate"> as having been successful.
 2: [RETRY] Retry ASDF operation.
 3: [CLEAR-CONFIGURATION-AND-RETRY] Retry ASDF operation after resetting the configuration.
 4: [RETRY] Retry ASDF operation.
 5: [CLEAR-CONFIGURATION-AND-RETRY] Retry ASDF operation after resetting the configuration.
 --more--

Backtrace:
  0: ("MOVZX" #<SB-ASSEM:SEGMENT {1006154583}> (:DWORD :QWORD) R9 #<SB-X86-64-ASM::EA  base=#<SB-C:TN t1[RCX] :LOAD> index=#<SB-C:TN t2[RAX] :NORMAL> disp=1>)
```
The top of the backtrace contains "MOVZX" which is mentioned in [SBCL release notes](http://www.sbcl.org/news.html#2.1.5) as minor incompatible change:
```
minor incompatible change: on x86-64, the backend instruction encoders for movzx and for string opcodes have changed their semantics. 
```

**Comment by 3b** [comment]
Update nibbles (and/or make sure you are actually loading it from where you intended). I think that was fixed in sharplispers/nibbles@dad2524

**Comment by nsrahmad** [comment]
Thanks, updating nibbles fixed it.
